src/plan_expand_hypertable.c

```diff
@@ -169,6 +169,7 @@ transform_time_bucket_comparison(PlannerInfo *root, OpExpr *op)
 	{
 		/* column < value + width */
 		Expr *subst;
```

Comment:
I think `subst` also needs to be declared volatile since it's used in both try and catch